dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -262,6 +263,15 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 		destroyableComponent->FixStats();
 	}
 
+	if (chatCommand == "server-version")
+	{
+		auto& serverName = Game::config->GetValue("server_name");
+		auto& serverVersion = Game::config->GetValue("server_version");
+		auto& gitCommitSha = Game::config->GetValue("git_commit_sha");
+		ChatPackets::SendSystemMessage(sysAddr, GeneralUtils::ASCIIToUTF16(serverName) + u" " + GeneralUtils::ASCIIToUTF16(serverVersion) + u" (" + GeneralUtils::ASCIIToUTF16(gitCommitSha) + u")");
```

Comment:
Don't use this function for text from a config file. It crashes when eg. `serverName` contains non-ascii characters.

Reply:
Yes you are right; I guess this is blocked then by #350 since there exists no conversion function for UTF-8 to UTF-16 yet?

Reply:
Technically it exists without that as std::codecvt_utf8_utf16.